index.html

```diff
@@ -167,6 +168,12 @@ <h3>Recommendation Track specification repositories</h3>
         <li><a href="https://github.com/w3c/ttml2/pulse">TTML2</a></li>
         <li><a href="https://github.com/w3c/ttml3/pulse">TTML3</a></li>
         <li><a href="https://github.com/w3c/webvtt/pulse">WebVTT</a></li>
+        <li><a href="https://github.com/w3c/adpt/pulse">ADPT</a></li>
+        <li><a href="https://github.com/w3c/tt-module-cjk-ext-1/pulse">TTML Extension Module for CJK</a></li>
```

Comment:
Please change link text to "TTML CJK Extension Module 1".

Reply:
Done.